src/widgets/widget_definitions.rs

```diff
@@ -148,12 +148,13 @@ pub(super) fn resolve_orientable_attrs(bargs: &mut BuilderArgs, gtk_widget: &gtk
 
 // concrete widgets
 
-/// @widget combo-box
-fn build_gtk_combo_box(bargs: &mut BuilderArgs) -> Result<gtk::ComboBoxText> {
+/// @widget combo-box-text
+/// @desc A combo box, where one can choose between several items (must be text, combo-box, with other widgets, is not yet implemented)
```

Comment:
I don't think it's necessary to mention "not yet implement" stuff in the docs here, as the API of the combo-box-text is quite obviously text-only

```suggestion
/// @desc A combo box allowing the user to choose between several items.
```

Reply:
hm tru